PxWeb/Code/Api2/ModelBinder/CommaSeparatedStringToListOfStrings.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace PxWeb.Code.Api2.ModelBinder
+{
+    public class CommaSeparatedStringToListOfStrings : IModelBinder
+    {
+        public Task BindModelAsync(ModelBindingContext bindingContext)
+        {
+
+            if (bindingContext == null)
+                throw new ArgumentNullException(nameof(bindingContext));
+
+            var modelName = bindingContext.ModelName;
+
+            var result = new List<string>();
+
+            var keys = bindingContext.HttpContext.Request.Query.Keys.Where(x => x.Equals(modelName, StringComparison.InvariantCultureIgnoreCase));
+
+            foreach (var key in keys)
+            {
+                string? q = bindingContext.HttpContext.Request.Query[key];
+                if (q != null)
+                {
+                    var items = Regex.Split(q, ",(?=[^\\]]*(?:\\[|$))", RegexOptions.IgnoreCase,
```

Comment:
Should we add try catch around the timeout add log something clever :-) 
?

Reply:
I think that we can let code higher up in the chain catch the error. I Changed to ReqexOptions.None instead no need for case insensitivity.